src/arc_utilities/ros_init.py

```diff
@@ -14,3 +18,45 @@ def shutdown():
     """ ensures the C++ node handle is shut down cleanly. It's good to call this a the end of any program
       where you called rospy_and_cpp_init """
     roscpp_initializer.shutdown()
+
+
+class RosContext:
+    def __init__(self, name):
+        rospy_and_cpp_init(name)
```

Comment:
Do we have to care about the `cpp_` node? If not, can you document that we should rarely interact with it directly. If so, can you log the node name to prevent having to look in the source to find it's prepended by `cpp_`?

Reply:
I don't often need to reference the name of the cpp node, but I'll add it to the doc string so one doesn't have to look through the code